OpenRA.Mods.Common/Traits/ExternalCapturable.cs

```diff
@@ -91,5 +91,15 @@ public void Tick(Actor self)
 			else
 				CaptureProgressTime++;
 		}
+
+		public bool PreventAutoTarget(Actor self, Actor attacker)
+		{
+			if (Captor == null)
+			{
+				return false;
+			}
+
+			return attacker.AppearsFriendlyTo(Captor);
```

Comment:
Could you change this entire block to `return Captor != null && attacker.AppearsFriendlyTo(Captor);`, please?